utils/lpac/files/lpac.sh

```diff
@@ -8,26 +8,29 @@ APDU_DEBUG="$(uci_get lpac global apdu_debug 0)"
 HTTP_BACKEND="$(uci_get lpac global http_backend curl)"
 HTTP_DEBUG="$(uci_get lpac global http_debug 0)"
```

Comment:
- Extra variables
- Default values are unneeded and shouldn't be used in the script here (because it's a code)

Reply:
Why are default values not needed?

I don't go with "extra variables, they do not propagate outside the script.

Reply:
Separating code and data is a good practice.
Additionally, the default values are just not needed here.

Please pay attention to the alternative implementation which doesn't use the defaults in code.

Reply:
> I don't go with "extra variables, they do not propagate outside the script.

Having a mix of the real Lpac's env vars and your extra (unneeded) ones makes the script harder to read.